.changeset/shy-files-jam.md

```diff
@@ -0,0 +1,34 @@
+---
+"@fluidframework/presence": minor
```

Comment:
I'm surprised not to see any API report updates - is this change not exported / user-facing? If not, we probably don't need a changeset entry. But if so, I would expect to see report changes.

Reply:
There are no external API changes - but internally the behavior has changed. Messages were formerly sent immediately all the time; now they'll be batched/throttled by default.